_sass/footer.scss

```diff
@@ -2,6 +2,22 @@
 
 footer {
 	color: $navy;
+
```

Comment:
Had to scope these styles within `footer` as they were impacting global scope.

Realizing now this file is indented with tabs while the other `.scss` files are indented with spaces. Should I use this PR to change the indentation here?

Reply:
I think added this change on this PR will be fine. I will send back with this request.